physical/raft/raft.go

```diff
@@ -905,14 +954,23 @@ func (b *RaftBackend) AddPeer(ctx context.Context, peerID, clusterAddr string) e
 	b.l.RLock()
 	defer b.l.RUnlock()
 
-	if b.raft == nil {
-		return errors.New("raft storage is not initialized")
+	if b.autopilot == nil {
```

Comment:
This was mainly done to accommodate for tests that use `getRaft` which basically creates an instance of raft backend and sets up a cluster and begins operating on them. But in the current code we have made such that we only instantiate autopilot if we are the active node, which isn't a thing when using `getRaft`. All those tests would fail telling that autopilot wasn't initialized.

This was one way to go about it.

An alternative is to have `SetupAutopilot` config in `raft.SetupOpts` and have SetupCluster do the initialization of autopilot if `SetupAutopilot` is true.

Another alternative is to disable autopilot in those tests explicitly.

Reply:
I actually came to see why having multiple causes to have autopilot disabled can cause reasoning troubles. For tests that operates on raft at the backend level only, having autopilot explicitly disabled seemed like the right way to go. Fixed. Thanks for the suggestion.